### PR TITLE
DOC-12605 Update GSI Rebalance Settings

### PR DESCRIPTION
**Pull request by rao-shwe**
DOC-12605

### CODE REVIEW

File: modules/manage/pages/manage-settings/general-settings.adoc [hunk 1]
```diff
@@ -318,7 +318,10 @@ The default cleared setting has an Index Server node rebuild any newly assigned
 You cannot enable file-based rebalance when you have enabled Memory Optimized Index Storage.  
 When you select this option, Couchbase Server copies the index files from one Index Server node to another during a rebalance instead of rebuilding them. 
 See xref:learn:clusters-and-availability/rebalance.adoc#index-rebalance-methods[Index Rebalance Methods].
-
++
+You can deactivate this feature from the UI or via REST API. To learn about deactivating this feature via REST API, see xref:rest-api:post-settings-indexes.adoc#deactivate-file-transfer-based-rebalance[Curl Command to Deactivate the File Transfer Based Rebalance Setting]. This feature is inactive by default.
++
+WARNING: Deactivating this feature slows down the Rebalance operation.
```
Comment (RayOffiah) [inline modules/manage/pages/manage-settings/general-settings.adoc]:
```suggestion
WARNING: Disabling this feature slows down the Rebalance operation.
```

File: modules/manage/pages/manage-settings/general-settings.adoc [hunk 2]
```diff
@@ -318,7 +318,10 @@ The default cleared setting has an Index Server node rebuild any newly assigned
 You cannot enable file-based rebalance when you have enabled Memory Optimized Index Storage.  
 When you select this option, Couchbase Server copies the index files from one Index Server node to another during a rebalance instead of rebuilding them. 
 See xref:learn:clusters-and-availability/rebalance.adoc#index-rebalance-methods[Index Rebalance Methods].
-
++
+You can deactivate this feature from the UI or via REST API. To learn about deactivating this feature via REST API, see xref:rest-api:post-settings-indexes.adoc#deactivate-file-transfer-based-rebalance[Curl Command to Deactivate the File Transfer Based Rebalance Setting]. This feature is inactive by default.
```
Comment (RayOffiah) [inline modules/manage/pages/manage-settings/general-settings.adoc]:
```suggestion
You can disable this feature from the UI or via REST API. To learn about disabling this feature via REST API, see xref:rest-api:post-settings-indexes.adoc#deactivate-file-transfer-based-rebalance[Curl Command to Deactivate the File Transfer Based Rebalance Setting]. This feature is disabled by default.
```

Comment (RayOffiah) [reply modules/manage/pages/manage-settings/general-settings.adoc]: A tiny point, but I think we tend to use `disable` rather than `deactivate`.

**Review by RayOffiah — CHANGES_REQUESTED** [review]

File: modules/rest-api/pages/post-settings-indexes.adoc [hunk 3]
```diff
@@ -155,3 +155,13 @@ include::example$post-settings-indexes.jsonc[]
 ----
 
 *401*:: This response code returns an empty body.
+
+[#deactivate-file-transfer-based-rebalance]
+== Curl Command to Deactivate the File Transfer Based Rebalance
```
Comment (RayOffiah) [inline modules/rest-api/pages/post-settings-indexes.adoc]:
```suggestion
Curl Command to Disable the File Transfer Based Rebalance
```

File: modules/rest-api/pages/post-settings-indexes.adoc [hunk 4]
```diff
@@ -155,3 +155,13 @@ include::example$post-settings-indexes.jsonc[]
...
+
+The following command deactivates the File Transfer Based Rebalance (`enableShardAffinity`) feature in the xref:manage:general-settings.adoc#file-based-index-rebalance[Advanced Index Settings].
```
Comment (RayOffiah) [inline modules/rest-api/pages/post-settings-indexes.adoc]:
```suggestion
The following command disables the File Transfer Based Rebalance (`enableShardAffinity`) feature in the xref:manage:general-settings.adoc#file-based-index-rebalance[Advanced Index Settings].
```

**Review by RayOffiah — CHANGES_REQUESTED** [review]
Minor quibbles.

File: modules/manage/pages/manage-settings/general-settings.adoc (hunk 2 above)
Comment (rao-shwe) [reply modules/manage/pages/manage-settings/general-settings.adoc]: Yes, Sarah too asked to follow what's on the UI.

**Comment by rao-shwe** [comment]
@RayOffiah 
Thank you for the suggestions.

For a few of your suggestions, a direct submission of "Commit suggestion" was greyed out, so I made the changes ('deactivate' to 'disable') in the editor and committed those changes.

**Review by RayOffiah — APPROVED** [review]
LGTM! 👍🏾

**Pull request merged**